src/routes/album/+page.svelte

```diff
@@ -1,135 +1,136 @@
 <script lang="ts">
-  import Fa from 'svelte-fa/src/fa.svelte'
-  import FaLayers from 'svelte-fa/src/fa-layers.svelte'
-  import { faX } from '@fortawesome/free-solid-svg-icons/faX'
-  import Gallery from '../../components/Gallery.svelte'
+	import Fa from 'svelte-fa/src/fa.svelte';
+	import FaLayers from 'svelte-fa/src/fa-layers.svelte';
+	import { faX } from '@fortawesome/free-solid-svg-icons/faX';
+	import Gallery from '../../components/Gallery.svelte';
 
-  let selectedPhoto: string = null
+	let selectedPhoto: string = null;
```

Comment:
It can be done in a further PR, but we should enable the [`strictNullChecks` flag](https://www.typescriptlang.org/tsconfig#strictNullChecks)